app/1.0.x/plugin-development/custom-entities.md

```diff
@@ -32,88 +36,99 @@ returns them.
 If you plugin is intended to support both Cassandra and PostgreSQL, then both
 migrations must be written.
 
-Each migration must bear a unique name, and `up` and `down` fields. Such fields
-can either be strings of SQL/CQL queries for simple migrations, or actual Lua
-code to execute for complex ones. The `up` field will be executed when Kong
-migrates **forward**. It must bring your database's schema to the latest state
-required by your plugin. The `down` field must execute the necessary actions to
-revert your schema to its previous state, before `up` was ran.
+If your plugin doesn't have it already, you should add a `<plugin_name>/migrations`
+folder to it. If there is no `init.lua` file inside. This is where all the migrations
```

Comment:
`If there is no `init.lua` file inside. `
-> 
`If there is no `init.lua` file inside, you should create one.` (needs to explain contents?)